primary_beam_correction.py

```diff
@@ -132,70 +123,62 @@ def standard_deviation(data):
     while diff > 0:
         i += 1
         if i > 50:
-            return sd
+            return 1/(sd)**2
         old_sd = sd
         old_len = len(data)
         cut = np.abs(data - med) < 5.0 * sd
         if np.all(~cut):
-            return sd
+            return 1/(sd)**2
         data = data[cut]
-        sd = mad(data)[1]
+        med, sd = mad(data)
         diff = old_len - len(data)
         if sd == 0.0:
             return old_sd
```

Comment:
This is still not fixed. I think you want `1/(old_sd)**2`

Reply:
I fixed others and probably missed it, but now is fixed.